go/integration/hcvault/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/google/tink/go/integration/hcvault
 go 1.12
```

Comment:
Please revert this change, as it's out of scope. You're welcome to send another PR for this.

Reply:
unfortunately my IDE keeps updating this automatically .. I will try to revet this outside my IDE

Reply:
changes reverted